Sortable.js

```diff
@@ -69,7 +69,7 @@
 		$ = win.jQuery || win.Zepto,
 		Polymer = win.Polymer,
 
-		captureMode = false,
+		captureMode = { capture: false, passive: true },
```

Comment:
No, that not correct.
1. For all browsers `captureMode` -> `false`
2. If a browser to support `passive`, only then we change `captureMode`.

Reply:
Okay, is there an easy way to know if the browser supports `passive`?

Reply:
Here it is checking, but instead of `true`, there was `false`, the Bug ;]
https://github.com/RubaXa/Sortable/pull/1202/files#diff-a8f06fcf61e5b94cb2e81ac878a55678R1488